src/components/Main/Main.tsx

```diff
@@ -42,6 +42,14 @@ export function severityErrors(severity: SeverityTableRow[]) {
 
 const severityDefaults: SeverityTableRow[] = updateSeverityTable(severityData)
 
+const isCountry = (country: string): country is keyof typeof countryAgeDistribution => {
```

Comment:
Interesting, never seen this syntax before.

Reply:
Yeah it's a not as well known TypeScript feature called type predicates. Super useful in some situations!